docs/user_guide/disconnected/index.md

```diff
@@ -56,7 +56,7 @@ single-layer DVD.
 
 Importing (step 3) can be done in one of two ways. The first way is to make the
 export available via HTTP to the importing Satellite instance. Simply put the
-export in `/var/www/pub/export`. After that, edit your CDN location from the
+export in `/var/www/html/pub/export`. After that, edit your CDN location from the
```

Comment:
This is a good point, the docs may need a more general going-over to account for usability tweaks that we added. I'll do that now and update this PR.